pilot/pkg/networking/core/v1alpha3/cluster_test.go

```diff
@@ -1808,6 +1808,94 @@ func TestInboundClustersPassThroughBindIPs(t *testing.T) {
 	}
 }
 
+func TestDualStackInboundClustersDefaultEndpoint(t *testing.T) {
+	dsProxy := model.Proxy{
+		Type:        model.SidecarProxy,
+		IPAddresses: []string{"1.1.1.1", "1111:2222::1"},
+		ID:          "v0.default",
+		DNSDomain:   "default.example.org",
+		Metadata: &model.NodeMetadata{
+			Namespace: "not-default",
+		},
+		ConfigNamespace: "not-default",
+	}
+
+	inboundFilter := func(c *cluster.Cluster) bool {
+		return strings.HasPrefix(c.Name, "inbound|")
+	}
+
+	cases := []struct {
```

Comment:
Should there be a case for when isDual is true but the defaultEndpoint is invalid?